src/main/java/beans/CommentBean.java

```diff
@@ -0,0 +1,75 @@
+package beans;
+
+public class CommentBean {
+	
+	public static final long serialVersionUID = 1L;
+	
+	private int userId;
+	private int commentatorId;
+	private boolean vote;
+	private String comment;
+	private String commentator;
+	
+	public CommentBean() {}
+	
+	public CommentBean(int userId,
+			boolean vote,
+			String comment,
+			int commentatorId,
+			String commentator) {
+		super();
+		this.userId = userId;
+		this.vote = vote;
+		this.comment = comment;
+		this.commentatorId = commentatorId;
+		this.commentator = commentator;
+	}
+	
+	private String resolveNull(String string) {
+		if(string == null) {
+			string = "";
+		}
+		return string;
+	}
```

Comment:
I would recommend extracting this method to something like BeanHelper which could be extended by all beans as this method is used in several places.